README/handlebars.md

```diff
@@ -0,0 +1,15 @@
+# Working with Handlebars
```

Comment:
I would also add the link to the handlebars website and/or npm package.

Also kind of cool, if you have the prettier extension installed (I assume it is needed, unless vscode has a built in default), but you can actually format readme files, and it will make all the spacing and indentation proper, etc.